example/style.css

```diff
@@ -22,10 +22,14 @@ body {
   right: -30rem;
 }
 
-.js-accordion [aria-hidden=true] {
+.glossary div[aria-hidden=true] {
   display: none !important;
 }
 
+.glossary button {
```

Comment:
Oh, this was just ugly basic styling for the example. Do you think it's still worth re-writing?